Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/ISISIndirectEnergyTransferWrapperTest.py

```diff
@@ -0,0 +1,313 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# pylint: disable=too-many-public-methods,invalid-name
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+from mantid.simpleapi import (CreateSimulationWorkspace, CreateWorkspace, CompareWorkspaces, DeleteWorkspace,
+                              ISISIndirectEnergyTransferWrapper)
+from mantid.api import WorkspaceGroup
+
+import numpy as np
+
+
+def _generate_calibration_workspace(instrument, ws_name='__calib'):
+    simulation_workspace = CreateSimulationWorkspace(Instrument=instrument,
+                                                     BinParams='0,1,2')
+    number_of_histograms = simulation_workspace.getNumberHistograms()
+
+    calib_workspace = CreateWorkspace(OutputWorkspace=ws_name,
+                                      DataX=np.ones(1),
+                                      DataY=np.ones(number_of_histograms),
+                                      NSpec=number_of_histograms,
+                                      ParentWorkspace=simulation_workspace)
+
+    DeleteWorkspace(simulation_workspace)
+
+    return calib_workspace
+
+
+class ISISIndirectEnergyTransferWrapperTest(unittest.TestCase):
+
+    def test_that_a_basic_reduction_will_result_in_an_output_group_containing_a_workspace_with_the_correct_name(self):
+        workspace = ISISIndirectEnergyTransferWrapper(InputFiles=['IRS26176.RAW'],
```

Comment:
we seem to have the same args every time, we could make this its own function and just return the value. It may even be worth placing it in the set up and having the output workspace as a member `self._workspace `

Reply:
The args change depending on the test (although the first three tests have the same args) so I think I will leave this as is for now